WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -0,0 +1,278 @@
+import Foundation
+import UIKit
+
+
+protocol WPRichContentViewDelegate: UITextViewDelegate
+{
+    func richContentView(richContentView: WPRichContentView, didReceiveImageAction image: WPRichTextImage)
+}
+
+
+/// A subclass of UITextView for displaying HTML formatted strings.  Embedded content
+/// in tags like img, iframe, and video, are loaded manually and presented as subviews.
+///
+class WPRichContentView: UITextView
+{
+    struct Constants {
+        static let photonQuality = 65
+        static let textContainerInset = UIEdgeInsetsMake(0.0, 0.0, -16.0, 0.0)
+        static let defaultAttachmentHeight = CGFloat(150.0)
+    }
+
+    /// Used to keep references to image attachments.
+    ///
+    var mediaArray = [RichMedia]()
+
+    /// Manages the layout and positioning of text attachments.
+    ///
+    lazy var attachmentManager: WPTextAttachmentManager = {
+        return WPTextAttachmentManager(textView: self, delegate: self)
+    }()
+
+    /// Used to load images for attachments.
+    ///
+    lazy var imageSource: WPTableImageSource = {
+        let source = WPTableImageSource(maxSize: self.maxDisplaySize)
+        source.delegate = self
+        source.forceLargerSizeWhenFetching = false
+        source.photonQuality = Constants.photonQuality
+        return source
+    }()
+
+    /// The maximum size for images.
+    ///
+    lazy var maxDisplaySize: CGSize = {
+        let bounds = UIScreen.mainScreen().bounds
+        let side = max(bounds.size.width, bounds.size.height)
+        return CGSize(width: side, height: side)
+    }()
+
+    /// Whether the view shows private content. Used when fetching images.
+    ///
+    var isPrivate = false
+
+    var content: String {
+        get {
+            return text ?? ""
+        }
+        set {
+            let str = newValue ?? ""
+            let style = "<style>" +
+                "body { font-family: Merriweather; font-size:16.0; line-height:1.6875; color: #2e4453; } " +
+                "a { color: #0087be; text-decoration: none; } " +
+                "a:active { color: #005082 } " +
+                "blockquote { font-style: italic; color:#4f748e; } " +
```

Comment:
I agree but this is probably going to change a bit when we update reader comments.  I'd like to hold off until we see exactly what needs we'll have there.